ds-collector/rust-commands/collect-info.rs

```diff
@@ -336,6 +336,18 @@ const COMMANDS: &[Cmd<'static>] = &[
         use_sudo: false,
         use_timeout: false,
     },
+    // cqlsh "$(hostname)" $cqlshOpts -e 'DESCRIBE FULL SCHEMA;' > "$artifactDir/schema.cql"
+    // this overrides schema.cql (but won't work in Cassandra verision 1.2 so is optional)
+    Cmd {
+        command: "cqlsh",
+        args: "{cqlsh_host} {cqlsh_port} {cqlsh_opts} -f {artifact_dir}/execute_full_schema.cql",
```

Comment:
this is not working. even when the cqlsh fails, the output file is clobbered empty